test/_init.test.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This is necessary because you can't await commands in index.ts as it's not an async function.  This basically just makes sure the extension is activated before we start trying to do any tests.

Reply:
So are you depending on this to get run first by the fact that "_init.test.ts" is alphabetized first?  Better would be to use a suiteSetup(), which can appear in any test file.

Reply:
I considered using suiteSetup, but I didn't want to have to do it per test file.  In that case, I had to call suiteSetup in the first test that would be run (which seemed to be in alphabetical order) and came to this solution.

Reply:
It's not per test file.

Reply:
And doesn't matter where it appears.

Reply:
Just to confirm, any suiteSetup() call will be executed before any test is run in all of the test files within the directory?

Reply:
Mocha documentation is horrible, but see https://github.com/Microsoft/vscode-docker/blob/145935f31ac58f2e409db23214e8280cde67845c/test/global.test.ts#L48 for example. suite()/test() etc. just register the tests they represent. Tests are not run until all top-level stuff is registered. So it doesn't matter when/where it's found, it will be run first, and you can have multiple of them.

Reply:
I just want to clarify that "top-level" is important in what Stephen is saying. For example:
```
suiteSetup1(...)

suite('example' () => {
    
    suiteSetup2(...)
})
```

suiteSetup1 will be run before everything. suiteSetup2 will only be run before the tests inside 'example'

Also don't necessarily use my existing test code as an example. I don't know if I knew that when originally writing these tests.